frontend/dockerfile/docs/reference.md

```diff
@@ -309,10 +309,10 @@ PS E:\myproject>
 
 ## Environment replacement
```

Comment:
I like the update, but it made me realize that we should rephrase this section a little bit more. "Environment variables" specifically means variables set in the system. Build arguments are never set on the system-level, they're interpreted when we're building.

I think `s/environment variables/variables/g` (within this section) would be an appropriate change to make. (And `## Variable replacement`, or maybe `## Variable substitution` for the section title.)